action.yml

```diff
@@ -366,8 +367,9 @@ runs:
                   echo "${wheelsurl}"
                   echo "✅ Python wheels" >> "${{ github.action_path }}/result"
               else
-                echo "❌ Python wheels (more-info: https://hacs.xyz/docs/publish/include#check-wheels" >> "${{ github.action_path }}/result"
-                touch "${{ github.action_path }}/error"
+                echo "⚠ Python wheels (more-info: https://hacs.xyz/docs/publish/include#check-wheels" >> "${{ github.action_path }}/result"
+                # NOTE: The following should not be a HARD failure, but a warning.
+                # touch "${{ github.action_path }}/error"
```

Comment:
This breaks the purpose of the action.
For your own repo, you can disable this check